functions.php

```diff
@@ -31,23 +31,28 @@ function displayChairs($query): string
 }
 
 /** create a function to insert the POST form data into the database
+ * @param PDO $db
 
-function insertChair()
-{
-    if(isset($_POST['submit']))
+ */
+
+function insertChair(PDO $db){
+
+    if(isset($_POST['submit'])
+        && (!empty($_POST['chairname']) && ($_POST['designer']) && ($_POST['designyear']) && ($_POST['imagelink'])))
```

Comment:
You'll need to wrap each of thee conditions in !empty(), currently, that check is only being applied to the chairname.

Reply:
I did that at first but then I thought the !empty might be able to cover all of them in one go and save on code.

I'll revert it to the way it was.

Cheers.